### PR TITLE
chore: add arrow keys click

### PR DESCRIPTION
**Pull request by SvyatK**

### CODE REVIEW

**Review by alfredwesterveld — APPROVED** [review]

File: packages/dullahan-adapter-selenium-3/src/DullahanAdapterSelenium3.ts [hunk 1]
```diff
@@ -333,6 +333,29 @@ export default class DullahanAdapterSelenium3 extends DullahanAdapter<DullahanAd
...
+                break;
+            case 'ArrowUp' :
+                await driver.actions().sendKeys(Key.ARROW_UP).perform();
+                break;
+            case 'ArrowDown' :
+                await driver.actions().sendKeys(Key.ARROW_DOWN).perform();
+        }
+
+    }
+
```
Comment (wilcokaartje2go) [inline packages/dullahan-adapter-selenium-3/src/DullahanAdapterSelenium3.ts:358]: We mainly (only?) use selenium 4. Please add the code the selenium4 adapter as well.

File: packages/dullahan-adapter-selenium-3/src/DullahanAdapterSelenium3.ts [hunk 2]
```diff
@@ -333,6 +333,29 @@ export default class DullahanAdapterSelenium3 extends DullahanAdapter<DullahanAd
         await driver.actions().sendKeys(keys).perform();
     }
 
+    public async pressKey(key: string): Promise<void> {
```
Comment (wilcokaartje2go) [inline packages/dullahan-adapter-selenium-3/src/DullahanAdapterSelenium3.ts:336]: Would be good to type key like:
type Key = 'ArrowLeft' | 'ArrowRight' | 'ArrowUp' | 'ArrowDown';

File: packages/dullahan-adapter-puppeteer/src/DullahanAdapterPuppeteer.ts [hunk 3]
```diff
@@ -244,6 +244,16 @@ export default class DullahanAdapterPuppeteer extends DullahanAdapter<DullahanAd
         await page.keyboard.type(keys);
     }
 
+    public async pressKey(key: string): Promise<void> {
```
Comment (wilcokaartje2go) [inline packages/dullahan-adapter-puppeteer/src/DullahanAdapterPuppeteer.ts:247]: Would be good to type key like:
type Key = 'ArrowLeft' | 'ArrowRight' | 'ArrowUp' | 'ArrowDown';

File: packages/dullahan-adapter-selenium-3/src/DullahanAdapterSelenium3.ts (hunk 1 above)
Comment (SvyatK) [reply packages/dullahan-adapter-selenium-3/src/DullahanAdapterSelenium3.ts:358]: done

**Review by wilcokaartje2go — APPROVED** [review]

**Pull request merged**